PyInstaller/loader/pyiboot01_bootstrap.py

```diff
@@ -122,22 +122,25 @@ def isatty(self):
     import ctypes
     import os
     from ctypes import LibraryLoader, DEFAULT_MODE
-
-    class PyInstallerCDLL(ctypes.CDLL):
-        def __init__(self, name, *args, **kwargs):
+    
+    def _frozen_name(name):
+        if name is not None:
```

Comment:
Hmm, would `name=''` be an valid input? I assume `if name:` would be a better test here.

(When changing this, please remove the trailing whitespace in line 125, too. Thanks.)